src/components/participants-list/participants-list.tsx

```diff
@@ -0,0 +1,43 @@
+import { Participant } from "@/app/events/[event]/participants/page";
+
+interface ParticipantListProps {
+  participants: Participant[];
+}
+
+export function ParticipantsList({ participants }: ParticipantListProps) {
+  return (
+    participants.length && (
+      <ul className="inline-flex mt-[12px]">
+        {participants.map((participant, index) => (
```

Comment:
Let's move this render logic to the place where we render the component to make it more flexible. See my comments about the `round-list.tsx` file.

Reply:
Done